source/errors.rst

```diff
@@ -54,7 +54,7 @@ disclosing sensitive information.
 
   {
     "status": 500,
-    "error": "Server error",
+    "error": "Server Error",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L44-L46 shows `'Server error'` is the correct form.